src/plot.js

```diff
@@ -221,9 +221,10 @@ export function plot(options = {}) {
     addScaleChannels(newChannelsByScale, stateByMark, options, (key) => newByScale.has(key));
     addScaleChannels(channelsByScale, stateByMark, options, (key) => newByScale.has(key));
     const newScaleDescriptors = inheritScaleLabels(createScales(newChannelsByScale, options), scaleDescriptors);
-    const newScales = createScaleFunctions(newScaleDescriptors);
+    const {scales: newIntantiatedScales, ...newScales} = createScaleFunctions(newScaleDescriptors);
```

Comment:
Typo “intantiated”, but also this is a new word for what we call an “exposed” scale elsewhere.